source/configuration_manual/virtual_attachments_plugin.rst

```diff
@@ -0,0 +1,54 @@
+.. _virtual_attchments_plugin:
+
+==========================
+Virtual Attachments Plugin
+==========================
+
+Virtual attachments
+===================
+
+First, you'll have to load the plugin which also requires the :ref:`virtual_plugin`
+to be loaded:
+
+.. code-block:: none
+
+  mail_plugins = $mail_plugins virtual virtual_attachments
+
+Namespace configuration
+=======================
+
+Then, you'll have to create a :ref:`namespaces` for the virtual-attachments,
+for example:
+
+.. code-block:: none
+
+  namespace {
+    prefix = virtual-attachments/
+    separator = /
+    location = virtual:~/virtual-attachments
+  }
+
+For every mailbox that shall get its own attachments folder, an according folder
+below that namespace needs to be created. The mailbox names need to match
+existing real or virtual mailboxes. Those will be mirrored below the
+virtual-attachments namespace then and contain virtual messages - one per
+attachment - based on the referenced original mailbox. The mailboxes can be
+auto created.
+
```

Comment:
the above paragraph could use bit work, it's very hard to read now.

Reply:
Better now?

Reply:
I think it's good, but could use an example at the bottom, like:

 * Attachments in INBOX are in virtual-attachments/INBOX
 * Attachments in virtual/All are in virtual-attachments/virtual/All

Also isn't there often the virtual/all autocreated? I'm not sure actually. But I think:

```
namespace {
  mailbox virtual/All {
    auto = create
  }
}
```

Reply:
Ok i added the examples, namespace section names and autocreated "All"  mailboxes.